Mathlib/Topology/Irreducible.lean

```diff
@@ -114,17 +116,40 @@ theorem isClosed_of_mem_irreducibleComponents (s) (H : s ∈ irreducibleComponen
   exact subset_closure.antisymm (H.2 H.1.closure subset_closure)
 #align is_closed_of_mem_irreducible_components isClosed_of_mem_irreducibleComponents
 
-theorem irreducibleComponents_eq_maximals_closed (X : Type*) [TopologicalSpace X] :
-    irreducibleComponents X = maximals (· ≤ ·) { s : Set X | IsClosed s ∧ IsIrreducible s } := by
+section IsIrreducibleClosed
+
+variable {S : Type*} [TopologicalSpace S] (s : Set S)
+
+/--
+A set `s : Set X` satisfies `IsIrreducibleClosed s` if `s` is irreducible and closed.
+-/
+def IsIrreducibleClosed : Prop := IsIrreducible s ∧ IsClosed s
```

Comment:
Now you have the `IrreducibleCloseds` structure, this `def` should probably be removed.